xcm/xcm-executor/src/traits/token_matching.rs

```diff
@@ -104,3 +105,18 @@ impl<AssetId, Instance> MatchesNonFungibles<AssetId, Instance> for Tuple {
 		Err(Error::AssetNotFound)
 	}
 }
+
+pub trait MatchesMultiAssetId<Id> {
```

Comment:
This isn't actually necessary -- you can simply use Contains<MultiLocation> to do the job, or even Get<MultiLocation>.